Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: all build build-rust build-go test
+.PHONY: all build build-rust build-go test format
```

```suggestion
.PHONY: all build build-rust build-go test
```

This is above the job definition already (which is the better notation as far as I can tell)